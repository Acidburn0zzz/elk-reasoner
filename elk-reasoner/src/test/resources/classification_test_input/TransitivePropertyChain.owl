Prefix( : = <http://example.org/> )
Prefix( owl: = <http://www.w3.org/2002/07/owl#> )
Ontology(<www.example.org#transitive_property_chain>
TransitiveObjectProperty(:T)
SubClassOf(:A ObjectSomeValuesFrom(:T :B))
SubClassOf(:B ObjectSomeValuesFrom(:T :C))
SubClassOf(:C ObjectSomeValuesFrom(:T :D))
SubClassOf(:D ObjectSomeValuesFrom(:T :E))
EquivalentClasses(:X ObjectSomeValuesFrom(:T :E))
)