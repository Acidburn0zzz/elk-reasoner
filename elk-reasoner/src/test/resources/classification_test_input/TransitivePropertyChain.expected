Prefix( : = <http://example.org/> )
Prefix( owl: = <http://www.w3.org/2002/07/owl#> )
Ontology(<www.example.org#transitive_property#taxonomy>
SubClassOf(:A :X)
SubClassOf(:B :X)
SubClassOf(:C :X)
SubClassOf(:D :X)
Declaration(Class(:E))

)